Dockerfile

```
@@ -8,9 +8,11 @@ RUN test -n "${VERSION}"
 
 RUN apk add -U make
 RUN make linux VERSION=${VERSION}
+RUN apk --no-cache add ca-certificates
```

```suggestion
RUN apk add -U make ca-certificates
RUN make linux VERSION=${VERSION}
```

We can do both installs in one line.